jsapp/js/components/account/dataStorage.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/_colors";
+
+.data-storage {
+  padding: 30px 40px;
```

Comment:
Please use the new way™ of using variables:

```
@use "scss/_colors"; //instead of @import
@use "scss/sizes";

color: colors.$kobo-white;

padding: sizes.$x30 sizes.$x40;
```

For sizes I've pushed to `beta` the file for you to use it wherever. If there is no size you need, try to use an existing one first - if it looks ok, don't add new one, if it's better with new one, add it :)

Reply:
Same as above